about.html

```diff
@@ -13,19 +13,12 @@
 </head>
 
 <body>
-  <!-- Mobile Nav -->
-  <button type="button" class="hamburger"><i class="fa-solid fa-bars"></i></button>
-  <div class="links-mobile">
-    <a href="/" class="links">Home</a>
-    <a href="./about.html" class="links">About</a>
-    <a href="./contact.html" class="links">Contact</a>
-    <!-- <a href="#" class="links">Subscription</a> -->
-  </div>
   <!-- Nav Here -->
+  <button type="button" class="hamburger"><i class="fa-solid fa-bars"></i></button>
   <nav class="navigation-desktop">
     <div class="logo">LOGO</div>
     <div class="links-desktop">
-      <a href="./" class="links">Home</a>
+      <a href="./index.html" class="links">Home</a>
```

Comment:
Actually, there's a reason for me leaving the href as
 `<a href="./" class="links">Home</a>`
with the "./"

This is because the main page/the root document is index.html. Many webservers try and find that page and set is as your home page, vercel probably does the same.

Now when you add "index.html" to the href....it redirects you to the same page but this time the url will look like this `https://anime-info-platform.vercel.app/index.html` 
making the url long

that is why it's omitted

Reply:
I think it's the same thing we're discussing below but on about/contact you can return to the "./" which means go to "index.html" using the "home" button

this "home" button is not available when you are searching for an anime